src/main/java/lotto/domain/lotto/LottoTickets.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain.lotto;
+
+import static lotto.utils.LottoConstants.*;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import lotto.utils.LottoConstants;
+
+public class LottoTickets {
+    private final List<Lotto> lottoTickets;
+
+    public LottoTickets(PurchasePrice price) {
+        this.lottoTickets = createLottoTickets(price);
+    }
+
+    private List<Lotto> createLottoTickets(PurchasePrice price) {
+        List<Lotto> tickets = new ArrayList<>();
+        int numberOfTickets = getNumberOfTickets(price);
+
+        while (numberOfTickets-- > 0) {
+            tickets.add(createNewLotto());
+        }
```

Comment:
취향의 차이라고는 생각되지만, 
```suggestion
        while (numberOfTickets > 0) {
            tickets.add(createNewLotto());
            numberOfTickets-- ;
        }
```
와 같이 분리하는 것이 더 가독성이 좋다고 생각합니다!